tests/common.go

```diff
@@ -402,6 +403,124 @@ const (
 	pxctlCDListCmd = "pxctl cd list"
 )
 
+const (
+	IBMHelmRepoName   = "ibm-helm-portworx"
+	IBMHelmRepoURL    = "https://raw.githubusercontent.com/portworx/ibm-helm/master/repo/stable"
+	IBMHelmValuesFile = "/tmp/values.yaml"
```

Comment:
@kphalgun-px I thought we agreed moving all common constants to separate constants.go file

Reply:
Initially, I created a constants.go file but later realized removing all constants from common.go would be beyond the scope of this PR. I will address this in another PR.